app/routes/home.js

```diff
@@ -1,3 +1,12 @@
 import Ember from 'ember';
 
-export default Ember.Route.extend({});
+export default Ember.Route.extend({
+  model: function() {
```

Comment:
How do you feel about adding a linter rule to encourage the method shorthand?

```js
{
  model() {

  }
}
```

I usually see that in Ember codebases and I think that, since this is supposed to be as canonical as possible, we should use the ES6 standard for this.